### PR TITLE
fix(frontend): temporary guest link

### LINKED ISSUE #1948: 🐛 [Bug] Guest Link cannot be stable
## 🐛 Bug

Guest Link cannot be stable between two Meetings, since a user can join any meeting on that table.
> https://stage1.app.dreammall.earth/join-table/1

The Guest link should be bound to the specific meeting in BBB. A suggestion is to use the internal meeting id.
>  'internal-meeting-id': 'b070c2de30140b9f77b7fa2318de762112e592c5-1725451926427'
> https://stage1.app.dreammall.earth/joinTable/b070c2de30140b9f77b7fa2318de762112e592c5-1725451926427

### PR DESCRIPTION
**Pull request by Mogge**
### Issues
- fixes #1948 

### Todo

- [X] None

### CODE REVIEW

File: frontend/src/pages/table/@id/__snapshots__/Page.test.ts.snap [hunk 1]
```diff
@@ -1697,7 +1697,6 @@ exports[`Table Page > renders 1`] = `
           <div
             class="flat-text-field d-flex flex-column align-center pa-4 w-100"
             data-v-b63bd396=""
-            modelvalue="[object Object]"
```
Comment (Mogge) [inline frontend/src/pages/table/@id/__snapshots__/Page.test.ts.snap:1700]: Why did that happen?

File: frontend/src/components/malltalk/settings/TableSettingsRoot.vue [hunk 2]
```diff
@@ -57,14 +58,28 @@ const pageContext = usePageContext()
...
-  return pageContext.routeParams?.id ? Number(pageContext.routeParams.id) : null
+  return pageContext.routeParams?.id ? pageContext.routeParams.id : ''
 })
 
 const tablesStore = useTablesStore()
 const isModeratorData = inject<IsModeratorInjection>(IsModeratorSymbol, {
   isModerator: ref(false),
 })
 
+const { getTables } = storeToRefs(tablesStore)
```
Comment (Mogge) [inline frontend/src/components/malltalk/settings/TableSettingsRoot.vue:69]: May be a refactor would be nice, to have an injection that provides the currently active table here. But this is out of scope of this PR

Comment (mahula) [reply frontend/src/components/malltalk/settings/TableSettingsRoot.vue:69]: Then please create a ticket about this refactoring suggestion.

Comment (Mogge) [reply frontend/src/components/malltalk/settings/TableSettingsRoot.vue:69]: https://github.com/dreammall-earth/dreammall.earth/issues/2745

**Review by ulfgebhardt — APPROVED** [review]
Works as expected for me locally
